packages/core/src/blocks/TableBlockContent/TableBlockContent.ts

```diff
@@ -46,14 +46,18 @@ const TableParagraph = Node.create({
 
   parseHTML() {
     return [
-      { tag: "td" },
       {
         tag: "p",
+        priority: 1000,
```

Comment:
curious about the changes in this file; why were they necessary and are they covered in the tests?